### DISCUSSION TITLE
Import MRT_ToolbarDropZone for renderTopToolbar

### DISCUSSION THREAD

**Discussion by ryjogo** [KevinVandy/material-react-table#249] [Ideas]
If we totally override renderTopToolbar, when we are loosing the dropzone, how can one import? Currently we can import buttons MRT_ShowHideColumnsButton and the button set with MRT_ToolbarInternalButtons.

**Comment by KevinVandy** [comment]
sure we can export that. Be on the lookout for v1.4.1 probably
